examples/todoapp/lib/base/data_sources/local/todo_local_data_source.dart

```diff
@@ -2,66 +2,57 @@ import 'dart:async';
 
 import 'package:realm/realm.dart';
 
-import '../../app/initialization/realm_instance.dart';
 import '../../models/errors/error_model.dart';
 import '../../models/todo_model.dart';
 
 class TodoLocalDataSource {
-  final RealmInstance _realmInstance;
+  final Realm _realmInstance;
   TodoLocalDataSource(this._realmInstance);
 
-  Stream<List<$TodoModel>> allTodos() => _realmInstance.realm
+  Stream<List<$TodoModel>> allTodos() => _realmInstance
```

Comment:
I believe it would be better to rename these usages to just `_realm`, currently it's a bit more confusing

Reply:
Done